src/fixit/cli.py

```diff
@@ -194,6 +205,38 @@ def fix(
     ctx.exit(exit_code)
 
 
+@main.command()
+@click.pass_context
+@click.option("--stdio", type=bool, default=True, help="Serve LSP over stdio")
+@click.option("--tcp", type=int, help="Port to serve LSP over")
+@click.option("--ws", type=int, help="Port to serve WS over")
+@click.option(
+    "--debounce-interval",
+    type=float,
+    default=0,
```

Comment:
Reference `LSPOptions.debounce_interval` so that the CLI shares the same default value?